x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/tree/EsqlNodeSubclassTests.java

```diff
@@ -562,7 +562,7 @@ private Object makeMap(Class<? extends Node<?>> toBuildClass, ParameterizedType
 
     private int randomSizeForCollection(Class<? extends Node<?>> toBuildClass) {
         int minCollectionLength = 0;
-        int maxCollectionLength = 10;
+        int maxCollectionLength = 8;
```

Comment:
setting this to 8 for now - I will follow up and improve this check so it's dependent on `toBuildClass`